x-pack/plugins/ingest_manager/server/services/package_config.ts

```diff
@@ -36,9 +39,37 @@ function getDataset(st: string) {
 class PackageConfigService {
   public async create(
     soClient: SavedObjectsClientContract,
+    callCluster: CallESAsCurrentUser,
     packageConfig: NewPackageConfig,
     options?: { id?: string; user?: AuthenticatedUser }
   ): Promise<PackageConfig> {
+    // Make sure the associated package is installed
```

Comment:
Can we do these requests in parallel?

Do we have to wait for `ensureInstalledPackage` to complete before calling `getPackageInfo`?